src/Haskell/Ide/Engine/Plugin/Pragmas.hs

```diff
@@ -0,0 +1,126 @@
+{-# LANGUAGE DeriveAnyClass    #-}
+{-# LANGUAGE DeriveGeneric     #-}
+{-# LANGUAGE OverloadedStrings #-}
+module Haskell.Ide.Engine.Plugin.Pragmas where
```

Comment:
Come to think of this this doesn't really involve GhcMod at all, so this module name should be fine